File: ProjectMapLanguageServer/ProjectMapSg/ContractModels/PageError.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ProjectMapSg.ContractModels
{
    public enum PageError
    {
        // SyntaxErrors = 0, // TODO: some errors is fixed by RoutingSg
        LocationNotMatchDefinition = 10,
        PageNameNotMatchRouteName = 20,
        MultiplePagesPerFile = 30,
        PageHasMultipleDefinitionParts = 40,
        MultiupleNamespacesPerFile = 50
    }
}

File: Extension/src/ProjectMapData/LanguageServerContract/PageError.ts
/** type of error on a page */
export enum PageError
{
    syntaxErrors = 0,
    locationNotMatchDefinition = 10,
    pageNameNotMatchRouteName = 20,
    multiplePagesPerFile = 30,
    pageHasMultipleDefinitionParts = 40,
    multiupleNamespacesPerFile = 50
}